R/read_extract_acute.R

```diff
@@ -1,15 +1,13 @@
 #' Read Acute extract
 #'
-#' @param year Financial year for the BOXI extract.
+#' @param year Year of BOXI extract
+#' @param file_path BOXI extract location
 #'
 #' @return a [tibble][tibble::tibble-package].
 #'
-#' @export
-read_extract_acute <- function(year) {
-  extract_acute_path <- get_boxi_extract_path(year = year, type = "Acute")
-
+read_extract_acute <- function(year, file_path = get_boxi_extract_path(year = year, type = "Acute")) {
```

Comment:
I think we want this one to still be exported.
```suggestion
#' @export
read_extract_acute <- function(year, file_path = get_boxi_extract_path(year = year, type = "Acute")) {
```